centreon/www/include/configuration/configGenerate/xml/moveFiles.php

```diff
@@ -334,6 +336,25 @@
                         }
                     }
                 }
+                /**
+                 * VMWare configuration
+                 */
+                if (count($listVmWareFile) > 0) {
```

Comment:
it's useless because it's a foreach after ;) At least, replace count by `!== []`

Reply:
Indeed, and we should only have one file so I should check that there is only one file